Kronos-Shifts-Connector/Microsoft.Teams.Shifts.Integration/Microsoft.Teams.Shifts.Integration.API/Controllers/OpenShiftRequestController.cs

```diff
@@ -415,158 +408,159 @@ internal async Task ProcessOpenShiftsRequests(string isRequestFromLogicApp)
+                                        await this.shiftMappingEntityProvider.SaveOrUpdateShiftMappingEntityAsync(
+                                            expectedShift,
+                                            rowKey,
+                                            monthPartition).ConfigureAwait(false);
 
-                                            // 7. Having the necessary Graph API call made here - to the approval endpoint.
-                                            var approvalMessageModel = new OpenShiftRequestApproval
-                                            {
-                                                Message = Resource.OpenShiftRequestApprovalMessage,
-                                            };
+                                        // 7. Having the necessary Graph API call made here - to the approval endpoint.
+                                        var approvalMessageModel = new OpenShiftRequestApproval
+                                        {
+                                            Message = Resource.OpenShiftRequestApprovalMessage,
+                                        };
 
-                                            var approvalMessageModelStr = JsonConvert.SerializeObject(approvalMessageModel);
-                                            var approvalHttpClient = this.httpClientFactory.CreateClient("ShiftsAPI");
-                                            approvalHttpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", allRequiredConfigurations.ShiftsAccessToken);
+                                        var approvalMessageModelStr = JsonConvert.SerializeObject(approvalMessageModel);
+                                        var approvalHttpClient = this.httpClientFactory.CreateClient("ShiftsAPI");
+                                        approvalHttpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", allRequiredConfigurations.ShiftsAccessToken);
 
-                                            // Send Passthrough header to indicate the sender of request in outbound call.
-                                            approvalHttpClient.DefaultRequestHeaders.Add("X-MS-WFMPassthrough", allRequiredConfigurations.WFIId);
-                                            using (var httpRequestMessage = new HttpRequestMessage(HttpMethod.Post, "teams/" + user.ShiftTeamId + "/schedule/openshiftchangerequests/" + entityToUpdate.RowKey + "/approve")
+                                        // Send Passthrough header to indicate the sender of request in outbound call.
+                                        approvalHttpClient.DefaultRequestHeaders.Add("X-MS-WFMPassthrough", allRequiredConfigurations.WFIId);
+                                        using (var httpRequestMessage = new HttpRequestMessage(HttpMethod.Post, "teams/" + user.ShiftTeamId + "/schedule/openshiftchangerequests/" + entityToUpdate.RowKey + "/approve")
```

Comment:
same here, template string would be nicer

Reply:
Done